<?xml version="1.0" encoding="UTF-8" ?>
<testsuites errors="0"
            failures="0"
            tests="0">

  <testsuite name="pageObject.createTweet"
    errors="0" failures="0" hostname="" id="" package="pageObject" skipped="0"
    tests="0" time="0.000" timestamp="">
  

  
    <system-err>
       âœ– Testing if element &lt;div.css-901oao.r-daml9f.r-1qd0xha.r-a023e6.r-16dba41.r-ad9z0x.r-bcqeeo.r-qvutc0&gt; contains text &#39;The username and password that you entered did not match our records. Please double-check and try again.&#39; in 5000ms - expected &#34;contains text &#39;The username and password that you entered did not match our records. Please double-check and try again.&#39;&#34; but got: &#34;does not contain &#39;The username and password that you entered did not match our records. Please double-check and try again.&#39;&#34; (5324ms)
    at Object.before (/home/adel/Desktop/Test/adel/Logininvalidemail.js:11:25)
    at new Promise (&lt;anonymous&gt;)
  TypeError: Logininvalidpassword.loginpassword is not a function
   - writing an ES6 async test case? - keep in mind that commands return a Promise; 
   - writing unit tests? - make sure to specify &#34;unit_tests_mode=true&#34; in your config.
       at Object.before (/home/adel/Desktop/Test/adel/Logininvalidpassword.js:11:28)
       at new Promise (&lt;anonymous&gt;)
    </system-err>
  

    

  
  </testsuite>
</testsuites>
